src/providers/ext.rs

```diff
@@ -19,6 +19,9 @@ pub trait InfoProviderExt: InfoProvider {
         addresses: &Option<Vec<String>>,
     ) -> Option<Vec<AddressEx>> {
         let addresses = addresses.as_ref()?;
+        if addresses.is_empty() {
+            return None;
```

Comment:
This is safe because `modules` was exposed as nullable already by the gw. Instead of `Some([])`, `None` seems more correct.

Reply:
Hmm do you really think it makes more sense? ... I mean I have no strong opinion, but I would like that the client teams agree to this.

Reply:
it's by the same token as the reasoning behind `0x0...0` is mapped into `null` I would say. There is no meaningful information or alternative meaning to having an empty list instead of `null`, AFAIK. I will double check this just to be totally sure.

Reply:
To give an update: the teams were notified and the behaviour was agreed on.